extensions/PhabBugz/lib/Feed.pm

```diff
@@ -107,7 +107,17 @@ sub feed_query {
         my $revision = Bugzilla::Extension::PhabBugz::Revision->new({ phids => [$object_phid] });
 
         if (!$revision->bug_id) {
-            $self->logger->debug("SKIPPING: No bug associated with revision");
+            if ($story_text =~ /\s+created\s+D\d+/) {
+                # If new revision and bug id was omitted, make revision public
+                $self->logger->debug("No bug associated with new revision. Marking public.");
+                $revision->set_policy('view', 'public');
+                $revision->set_policy('edit', 'users');
+                $revision->update();
+                $self->logger->info("SUCCESS");
+            }
+            else {
+                $self->logger->debug("SKIPPING: No bug associated with revision change");
```

Comment:
While you probably don't log the whole story text, I can't help but feel if this breaks, we'd appreciate more details about what went wrong.